src/docs/slot_specifications.md

```diff
@@ -0,0 +1,317 @@
+
+### 2.5 (Structured comment) name abbreviation
+
+The term (slot) structured comment name (`name`) SHOULD be a abbreviated form of the item (title) attribute.
+
+Examples:
+
+| Term Item / `title`                             | Structured comment name / `name` |
+| ----------------------------------------------- | -------------------------------- |
+| geographic location (country and/or sea,region) | `geo_loc_name`                   |
+| isolation and growth condition                  | `isol_growth_condt`              |
+| pcr conditions                                  | `pcr_cond`                       |
+| sample volume or weight for DNA extraction      | `samp_vol_we_dna_ext`            |
+| collection site geographic feature              | `coll_site_geo_feat`             |
+
+### 2.6 (Structured comment) name common prefix
+
+When related to existing terms, the term (slot) structured comment name (`name`) SHOULD use a common prefix that allow grouping of related terms.
+
+Examples:
+
+- Terms related to `sample` should use the prefix `samp_`.
+
+  | Term Item / `title`              | Structured comment name / `name` |
+  | -------------------------------- | -------------------------------- |
+  | sample storage temperature       | `samp_store_temp`                |
+  | sample storage duration          | `samp_store_dur`                 |
+  | sample volume for DNA extraction | `samp_vol_we_dna_ext`            |
+
+- Terms related to assembly metadata term (slots) should use the prefix `assembly_`.
+
+  | Term Item / `title`          | Structured comment name / `name` |
+  | ---------------------------- | -------------------------------- |
+  | name and version of assembly | `assembly_name`                  |
+  | assembly software            | `assembly_software`              |
+  | assembly quality             | `assembly_qual`                  |
+
+## 3. Term expected value types
+
+### 3.1 Term expected value must be valid LinkML range types
+
+The type of data specified in the expected value (`range`) of a slot (term) MUST be in the form of a valid LinkML `range` type:
+
+- `string`
+- `integer`
+- `float`
+- `boolean`
+- A MIxS defined enumeration
+
+Refer to LinkML documentation for more information on [range types](https://linkml.io/linkml-model/latest/docs/range/).
+
+## 4. Slot attributes
+
+### 4.1. Minimal required LinkML slot attributes
+
+A term (slot) MUST at a minimum include following attributes:
+
+- [`description`](https://linkml.io/linkml/schemas/metadata.html#providing-descriptions).
+- [`title`](https://linkml.io/linkml-model/latest/docs/title/).
+- [`examples`](https://linkml.io/linkml-model/latest/docs/examples/).
+- [`in_subset`](https://linkml.io/linkml-model/latest/docs/in_subset/).
+- [`keywords`](https://linkml.io/linkml-model/latest/docs/keywords/).
+- [`slot_uri`](https://linkml.io/linkml-model/latest/docs/slot_uri/).
+- [`range`](https://linkml.io/linkml/schemas/slots.html#ranges).
+
```

Comment:
two points on section 4.1:
a. Should hte structured comment name be included in the list of required attributes? technically I guess its not an attribute in LinkML speak because its the slot identifier, but it is required.
b. For completeness can we include the MIxS terminology in the list as well?